src/etheroll/roll.kv

```diff
@@ -17,15 +17,15 @@
         size_hint_y: None
         MDTextField:
             id: bet_size_input_id
-            text: "0.1"
+            text: "0.01"
```

Comment:
I think the minimum bet is currently `0.1` rather than `0.01`. So perhaps we should leave the default to `0.1`